docs/en/quickstart/rest_api.md

```diff
@@ -95,7 +99,11 @@ The response:
 ```json
 {
     "code":0,
-    "msg":"ok"
+    "msg":"ok",
+    "result": {

```

Comment:
we should keep it consistent with [feature extract api](https://github.com/4paradigm/OpenMLDB/blob/44ddfde5e9a0edae6c23b46a286ea477091b3a67/docs/en/quickstart/rest_api.md#real-time-feature-extraction), use `"data"` as filed name